cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,62 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
+describe('Article flow', () => {
+  let user;
+  let article;
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+      cy.login(user.email, user.username, user.password);
+      cy.getCookie('auth').should('exist');
+      cy.reload();
+      cy.task('generateArticle').then((generatedArticle) => {
+        article = generatedArticle;
+      });
+    });
   });
 
-  it('', () => {
+  it('should allow user to create an article', () => {
+    cy.contains('.nav-link', 'New Article').click();
+    cy.url().should('contain', '/editor');
+
+    cy.findByPlaceholder('Article Title').type(article.title);
```

Comment:
The `cy.findByPlaceholder` method is not a standard Cypress command. You might need to install and configure the `cypress-testing-library` to use this method, or replace it with `cy.get('input[placeholder="Article Title"]')`.